.circleci/config.yml

```diff
@@ -76,6 +76,23 @@ jobs:
           command: |
             docker run panda_build /bin/bash -c "cd /panda/boardesp; make user1.bin"
 
+  safety_replay:
+    machine:
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - run:
+          name: Build image
+          command: "docker build -t panda_safety_replay -f tests/safety_replay/Dockerfile ."
+      - run:
+          name: Replay drives
+          command: |
+            mkdir /tmp/safety_replay
+            docker run -v /tmp/safety_replay:/tmp/safety_replay panda_safety_replay /bin/bash -c "cd /openpilot/panda/tests/safety_replay; PYTHONPATH=/openpilot ./test_safety_replay.py > /tmp/safety_replay/output.txt"
+      - store_artifacts:
+          name: Store safety_replay test output
+          path: /tmp/safety_replay/output.txt
```

Comment:
output is small, so we can just print it in the terminal. No need to store in an artifact